BoxStation: Secure Armory re-mapped

DogeDogIs opened the pull request:
Secure Armory was re-mapped and the non-secure armory was removed and Gamma Shuttle now docks with the Secure Armory.

<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->
The Secure Armory now have all items from non-secure Armory, and Gamma shuttles now docks directly with the Secure Armory. The non-secure armory was removed to make place for Gamma Shuttle.

## What Does This PR Do
<!-- Include a small to medium description of what your PR changes. Document all changes, as not doing this may delay reviews or even discourage maintainers from merging your PR! -->
This PR remakes the Secure Armory and removes the non-secure Armory, for better mapping, since all items inside the unsecured Armory were of good importance and at the same time, repositioning the Gamma shuttle to a better location, connected to the Armory, where it might make more sense.

## Why It's Good For The Game
<!-- Add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->
The change is a benefit to the game as at the same time it makes the important items move, at the same time, the Gamma shuttle will go to a place that makes more sense to be connected, along with the Secure Armory. Placing some implants together with the Secure Armory also helps the Security Officers' work, making them not waste time for the AI ​​to open up to take, for example, a tracking implant for perma prisoners.

## Images of changes
<!-- If you did not make a map or sprite edit, you may delete this section. You may include a gif or mp4 of your feature if you want. -->
![armory4](https://user-images.githubusercontent.com/62888630/179573413-807e42dc-107c-4840-ab71-0142dd6297bf.png)
![armory](https://user-images.githubusercontent.com/62888630/179573421-8fd6c81a-54f5-45f3-82b8-50080fbdc955.png)
![armory2](https://user-images.githubusercontent.com/62888630/179573422-39a8027b-d12d-4fe2-a401-453e0c9ed895.png)
![armory3](https://user-images.githubusercontent.com/62888630/179573424-177dcebc-90e9-474a-a530-5a7e3d95ea1a.png)


## Changelog
:cl:
add: Added new location for gamma shuttle to dock with.
del: Removed Non-secure Armory.
tweak: Remapped the Secure Armory and Warden window.

/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->
<!-- If a PR has no impact on players (i.e. a code refactor that does not change functionality) then the entire Changelog heading and contents can be removed. -->

Inadvizable commented:
This is very cool, but I think the implants should go in warden's office. Otherwise officers will do the warden's job without even asking.

DogeDogIs commented:
> This is very cool, but I think the implants should go in warden's office. Otherwise officers will do the warden's job without even asking.

Secs officers do that even if is locked in a room, they ask AI to open. Sometimes Warden is busy too, this can be good or bad, it depends on the situation. Imagine if you don't have a AI, Warden is dead and HoS is dead, the time you will save from having them in the Armory is very good.

DogeDogIs commented:
> Secure Armory was re-mapped and the non-secure armory was removed and Gamma Shuttle now docks with the Secure Armory.
> 
> The Secure Armory now have all items from non-secure Armory, and Gamma shuttles now docks directly with the Secure Armory. The non-secure armory was removed to make place for Gamma Shuttle.
> 
> ## What Does This PR Do
> This PR remakes the Secure Armory and removes the non-secure Armory, for better mapping, since all items inside the unsecured Armory were of good importance and at the same time, repositioning the Gamma shuttle to a better location, connected to the Armory, where it might make more sense.
> 
> ## Why It's Good For The Game
> The change is a benefit to the game as at the same time it makes the important items move, at the same time, the Gamma shuttle will go to a place that makes more sense to be connected, along with the Secure Armory. Placing some implants together with the Secure Armory also helps the Security Officers' work, making them not waste time for the AI ​​to open up to take, for example, a tracking implant for perma prisoners.
> 
> ## Images of changes
> ![armory4](https://user-images.githubusercontent.com/62888630/179573413-807e42dc-107c-4840-ab71-0142dd6297bf.png) ![armory](https://user-images.githubusercontent.com/62888630/179573421-8fd6c81a-54f5-45f3-82b8-50080fbdc955.png) ![armory2](https://user-images.githubusercontent.com/62888630/179573422-39a8027b-d12d-4fe2-a401-453e0c9ed895.png) ![armory3](https://user-images.githubusercontent.com/62888630/179573424-177dcebc-90e9-474a-a530-5a7e3d95ea1a.png)
> 
> ## Changelog
> 🆑 add: Added new location for gamma shuttle to dock with. del: Removed Non-secure Armory. tweak: Remapped the Secure Armory and Warden window.
> 
> /🆑
